display/display.go

```diff
@@ -352,6 +353,42 @@ func Init(version, commit, builtBy string) {
 			case config.CmdAddSub:
 				go addSubscription()
 				return nil
+			case config.CmdCopyPageURL:
+				currentURL := tabs[curTab].page.URL
+				err := clipboard.WriteAll(currentURL)
+				if err != nil {
+					Error("Copy Error", err.Error())
+					return nil
+				}
+				return nil
+			case config.CmdCopyTargetURL:
+				currentURL := tabs[curTab].page.URL
+				selectedURL := tabs[curTab].HighlightedURL()
```

Comment:
You should handle the case where `selectedURL` is an empty string, indicating nothing is selected. I would just do nothing in that case, `return nil`.